src/tasks/FishingTask.py

```
@@ -0,0 +1,390 @@
+            return False
+        zone_left = int(state.get("zone_left", 0))
+        zone_right = int(state.get("zone_right", 0))
+        pointer_center = int(state.get("pointer_center", -1))
+        image_width = max(1, int(state.get("image_width", 1)))
+        zone_width = max(0, int(state.get("zone_width", zone_right - zone_left)))
+        ratio = zone_width / image_width
+        if not (0.05 <= ratio <= 0.55):
+            return False
+        if not (0 <= pointer_center < image_width):
+            return False
+        # 过滤明显误检：绿区贴边且指针又远离，通常不是有效拉力条
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)